api/policy/policy.go

```diff
@@ -0,0 +1,124 @@
+package policy
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"strings"
+
+	"github.com/pkg/errors"
+
+	"github.com/CircleCI-Public/circleci-cli/api/header"
+	"github.com/CircleCI-Public/circleci-cli/settings"
+	"github.com/CircleCI-Public/circleci-cli/version"
+)
+
+// Client communicates with the CircleCI policy-service to ask questions
+// about policies. It satisfies policy.ClientInterface.
+type Client struct {
+	token     string
+	serverUrl string
+	client    *http.Client
+}
+
+type errorResponse struct {
```

Comment:
wrong error struct... Policy-Service returns errors in this format:

```go
type HTTPError struct {
        Error   string                 `json:"error"`
        Context map[string]interface{} `json:"context,omitempty"`
}
```

Reply:
Right!
How do you feel about not parsing the error response at all, and just return the raw response body in case of error?

Reply:
seems ok. You could parse out the error string and simply print the string rather than the entire json. Might be nicer visually, but that being said, I am 100% ok with raw json as well.